src/app/components/HoldConfirmation/HoldConfirmation.jsx

```diff
@@ -25,10 +25,50 @@ class HoldConfirmation extends React.Component {
 
   componentDidMount() {
     this.requireUser();
-
     document.getElementById('confirmation-title').focus();
   }
 
+// eligibilityErrorText supplies the appropriate text in case a patron is ineligible to place holds
```

Comment:
Align this comment and period at the end.

Reply:
I've rewritten this with esdoc